fpdf/html.py

```diff
@@ -245,6 +245,8 @@ def __init__(
         self.table_row_height = 0
         self.heading_level = None
         self.heading_sizes = dict(**DEFAULT_HEADING_SIZES)
+        self.heading_above = 0.2  # extra space above heading, relative to font size
+        self.heading_below = 0.2  # extra space below heading, relative to font size
```

Comment:
I know that this PR has been merged more than 20 months ago,
but I do not think that those properties have been documented so far.

@gmischler: would you agree to either:
* convert those properties into class properties?
* allow those properties to be provided to the class constructor?

...and to document those configurations settings in the `docs/`.

This way, we would gain in terms of consistency of the configuration options available to configure HTML rendering.

Reply:
Your strange use of terminology makes it a bit of a challenge to figure out what you really want to do... I somehow doubt it really has anything to do with [properties](https://docs.python.org/3/library/functions.html#property). :wink:

If you are suggesting to make those instance attributes configurable via constructor parameters, then I'm all for it. 👍 

I don't really see a point to turn them into class attributes, though.